src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,52 @@
+import { Dispatch, SetStateAction, useContext, useState } from "react";
+import { Theme } from "../../pages";
+import { ThemeContext } from "@/context/context";
+
+
+interface NavbarProps {
+  toggleFunction: Dispatch<SetStateAction<Theme>>;
+}
+
+
+function changeMode(): void {}
+function NavBar({ toggleFunction }: NavbarProps) {
+  let currentTheme = useContext(ThemeContext);
+  function changeMode(): void {
+    if (currentTheme == "Dark Mode") {
+      toggleFunction("Light Mode");
+    } else {
+      toggleFunction("Dark Mode");
+    }
+  }
+  
+  return (
+    <div
+      className={`app__navbar ${
+        currentTheme == "Dark Mode" ? "container_dark" : "container_light"
+      }`}
+    >
+      <p
+        className={`app__navbar-title ${
+          currentTheme == "Dark Mode" ? "dark" : "light"
+        }`}
+      >
+        Where in the world?
+      </p>
+      <div
+        className={`app__navbar-mode ${
+          currentTheme == "Dark Mode" ? "dark" : "light"
+        }`}
+        onClick={changeMode}
+      >
+        {currentTheme == "Dark Mode" ? (
+          <i className="fa-solid fa-moon"></i>
+        ) : (
+          <i className="fa-solid fa-sun"></i>
+        )}
+
+        <p>{currentTheme}</p>
+      </div>
+    </div>
+  );
+}
+export default NavBar;
```

Comment:
Please include screenshots in the comments so we can see how these look

Reply:
![image](https://github.com/Stage-Research-Center-Jake-Melvin/rest_countries_api/assets/138882910/0e04accd-b69e-4ef6-8ddb-c942815c8136)
![image](https://github.com/Stage-Research-Center-Jake-Melvin/rest_countries_api/assets/138882910/0e04accd-b69e-4ef6-8ddb-c942815c8136)
![image](https://github.com/Stage-Research-Center-Jake-Melvin/rest_countries_api/assets/138882910/0e04accd-b69e-4ef6-8ddb-c942815c8136)
![image](https://github.com/Stage-Research-Center-Jake-Melvin/rest_countries_api/assets/138882910/d010cff9-1c06-4e41-b08f-e58dc90687f5)
![image](https://github.com/Stage-Research-Center-Jake-Melvin/rest_countries_api/assets/138882910/d010cff9-1c06-4e41-b08f-e58dc90687f5)
![image](https://github.com/Stage-Research-Center-Jake-Melvin/rest_countries_api/assets/138882910/d010cff9-1c06-4e41-b08f-e58dc90687f5)

Reply:
By Screenshots, I meant Screenshot of the user interface that was coded